xsd/netex_part_3/part3_fares/netex_fareTable_version.xsd

```diff
@@ -91,8 +91,7 @@
 					<Requires>http://www.netex.org.uk/schemas/1.0/PATH/netex_prereqfile.xsd</Requires>
 				</Relation>
 				<Rights>Unclassified
-					 <Copyright>CEN, Crown Copyright 2009-2020</Copyright>
-				</Rights>
+					 <Copyright>CEN, Crown Copyright 2009-2020</Copyright></Rights>
```

Comment:
Try to avoid changing unrelated stuff.

Reply:
XMLSpy did do it automatically. Sorry, that I missed it.